src/framework/actions/internal/actionsdispatcher.cpp

```diff
@@ -48,6 +48,15 @@ void ActionsDispatcher::dispatch(const ActionCode& actionCode)
 
 void ActionsDispatcher::dispatch(const ActionCode& actionCode, const ActionData& data)
 {
+    // is query?
+    if (actionCode.find("://") != std::string::npos) {
+        IF_ASSERT_FAILED(data.empty()) {
+            LOGE() << "not supported action data with query";
+        }
+        dispatch(ActionQuery(actionCode));
+        return;
+    }
```

Comment:
what do you think about writing it this way so as not to check for "://" ?

```suggestion
    ActionQuery query = ActionQuery(actionCode);
    if (query.isValid()) {
        IF_ASSERT_FAILED(data.empty()) {
            LOGE() << "not supported action data with query";
        }
        dispatch(query);
        return;
    }
```

Reply:
good suggestion, agreed, changed